api.raml

```diff
@@ -23,28 +22,79 @@ securitySchemes:
           type: string
 
 
-#
-# Example Endpoints
-#
+/sequenceForGff3/{sequenceType}:
```

Comment:
You are using different paths because there are different possible input formats.  We should discuss that.  I don't know if that is proper REST.

Reply:
It's bad REST, also because all the other inputs for this are served as query parameters! I tried to make it work like that, so that the main part of the request - the .gff3 file - takes up the whole body of the POST, because I didn't know how to send a REST with a file which is part of a JSON payload. But actually, this didn't work either (I couldn't work out how to send a good POST with `curl`).

Reply:
in a meeting with wojtek we agreed to have the endpoint be just `/sequences`